### ISSUE TITLE
Watchdog is not configured if using wheezy

### ISSUE THREAD

**Issue by FooDeas** [FooDeas/raspberrypi-ua-netinst#56]
If wheezy is selected as release, the hardware watchdog is not configured.
Furthermore the Device Tree parameter (`dtparam=watchdog=on`) should be used instead of loading the module manually.

**Comment by FooDeas** [comment]
# devel notes
## common
## wheezy
- install watchdog package
- add Device Tree parameter (`dtparam=watchdog=on`) to config.txt~~ **Note:** test if module is loaded
- uncomment/add in `/etc/watchdog.conf`:
```
#max-load-1		= 24
#watchdog-device	= /dev/watchdog
watchdog-timeout = 14
```
## jessie
~~- add Device Tree parameter (`dtparam=watchdog=on`) to config.txt~~ **Note:** not needed
- no further changes needed

**Comment by FooDeas** [comment]
Closed with af43a6d07ad110a4e56d89b605ff083479e5c41a.
